roar-rails.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency "roar",          '~> 1.1.0'
+  s.add_runtime_dependency "roar",          "~> 1.1"
```

Comment:
@jandudulski let's update to 1.2 ?

Reply:
Yes please :rocket:

Reply:
Can you apply the suggestion please ? Seems GH doesn't have an option from UI to do so!

Reply:
@yogeshjain999 done